lib/get-soundcloud-url.js

```diff
@@ -0,0 +1,22 @@
+const Request = require('sdk/request').Request;
+const clientId = require('../package.json').config['SOUNDCLOUD_CLIENT_ID'];
+
+module.exports = getSoundcloudUrl;
+
+function getSoundcloudUrl(videoUrl, cb) {
+  Request({
+    url: `https://api.soundcloud.com/resolve.json?client_id=${clientId}&url=${videoUrl}`,
+    onComplete: function (resp) {
+      if (!resp.json) {
+        cb('Could not connect to soundcloud.com');
+        return;
+      }
+
+      if (resp.json.kind !== 'track' || !resp.json.streamable) {
+        cb('not track or not streamable');
```

Comment:
This currently only sets the error to TRUE when sent down to the client, it's more or less a placeholder until we add more specific error messaging. I've created https://github.com/meandavejustice/min-vid/issues/532 to track these types of instances